.github/workflows/test.yaml

```diff
@@ -60,14 +52,6 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Cache Docker layers
-        uses: actions/cache@v3
-        with:
-          path: /tmp/.buildx-cache
-          key: ${{ runner.os }}-buildx-${{ github.sha }}
-          restore-keys: |
-            ${{ runner.os }}-buildx-
-
```

Comment:
Have I missed something or is this an error? We're only concerned about `setup-go` action, right?

Reply:
😮 Yes!

Reply:
Fixed, thanks for catching this.